packages/components/tag/src/tag.stories.ts

```diff
@@ -1,292 +1,121 @@
-      table: {
-        disable: true
-      }
-    }
-  },
-  render: ({ disabled, emphasis, label, readonly, size }) => {
-    return html`
-      <style>
-        div {
-          inline-size: 200px;
-          display: flex;
-          flex-direction: column;
-          gap: 16px;
-        }
-      </style>
-      <div>
-        <sl-tag .emphasis=${emphasis} ?disabled=${disabled} ?readonly=${readonly} label=${label} .size=${size}></sl-tag>
-        <sl-tag
-          .emphasis=${emphasis}
-          ?disabled=${disabled}
-          ?readonly=${readonly}
-          label=${label}
-          removable
-          .size=${size}
-        ></sl-tag>
-      </div>
-    `;
+    ...Basic.args,
+    disabled: true
   }
 };
 
-export const List: Story = {
-  args: {
-    size: 'md',
-    label: 'Tag label',
-    removable: false,
-    emphasis: 'subtle'
-  },
-  render: ({ disabled, emphasis, label, readonly, removable, size }) => {
-    const amount = Array.from({ length: 50 });
-    return html`
-      <sl-tag-list .emphasis=${emphasis} .size=${size}>
-        ${amount.map(
-          () => html`
-            <sl-tag .label=${label} ?disabled=${disabled} ?removable=${removable} ?readonly=${readonly}></sl-tag>
-          `
-        )}
-        <sl-tag
-          label="I am a veeeeeeeeeeeeeeery long tag label"
-          ?disabled=${disabled}
-          ?removable=${removable}
-          ?readonly=${readonly}
-        ></sl-tag>
-        <sl-tag label="I am a label" ?disabled=${disabled} ?removable=${removable} ?readonly=${readonly}></sl-tag>
-        <sl-tag label="I am another label" ?disabled=${disabled} ?removable=${removable} ?readonly=${readonly}></sl-tag>
-      </sl-tag-list>
-    `;
-  }
-};
-
-export const Stacked: Story = {
+export const Overflow: Story = {
   args: {
-    size: 'md',
-    label: 'Tag label',
-    removable: true,
-    emphasis: 'subtle'
-  },
-  argTypes: {
-    disabled: {
-      table: {
-        disable: true
-      }
-    }
-  },
-  render: ({ emphasis, label, readonly, removable, size }) => {
-    const amount = Array.from({ length: 20 });
-    return html`
-      <style>
-        sl-tag-list {
-          margin-bottom: 24px;
-        }
-      </style>
-      <sl-tag-list stacked .emphasis=${emphasis} .size=${size}>
-        <sl-tag .emphasis=${emphasis} label="my label" ?removable=${removable} ?readonly=${readonly}></sl-tag>
-        <sl-tag .emphasis=${emphasis} label="test" ?removable=${removable} ?readonly=${readonly}></sl-tag>
-        <sl-tag
-          .emphasis=${emphasis}
-          label="This is a very long label of the tag"
-          ?removable=${removable}
-          ?readonly=${readonly}
-        ></sl-tag>
-        ${amount.map(
-          (_, index) => html`
-            <sl-tag .label=${label + ' ' + (index + 1)} ?removable=${removable} ?readonly=${readonly}></sl-tag>
-          `
-        )}
-      </sl-tag-list>
-    `;
+    label: 'This is a very long label which overflows',
+    maxWidth: '200px'
```

Comment:
When I change the maxWidth in the story, even when there is ellipsis visible, the tooltip is no longer showing on hover/focus

<img width="1588" alt="image" src="https://github.com/user-attachments/assets/29885e9e-10b5-40ad-8ee3-0747f4f0e2f6">

Reply:
I cannot reproduce that at this time.

Reply:
I think when I changed the `maxWidth` few times there in options it happened.